boards/Protectli/VP66xx/v0.9.0-results.csv

```diff
@@ -0,0 +1,192 @@
+Recovery boot popup confirmation (firmware),VBO010.001,FAIL,PASS
+Recovery popup is not displayed when correctly signed firmware is flashed in RW_A,VBO011.001,FAIL,PASS
+Self-signed binary is bootable without errors,VBO012.001,FAIL,PASS
+BIOS lock support (Ubuntu),BLS001.001,FAIL,PASS
+BIOS lock support deactivation (Ubuntu),BLS002.001,FAIL,PASS
+Measured Boot support (Ubuntu),MBO001.001,FAIL,PASS
+Check Secure Boot default state (firmware),SBO001.001,PASS,PASS
+UEFI Secure Boot (Ubuntu),SBO002.001,PASS,PASS
+UEFI Secure Boot (Windows),SBO002.002,PASS,PASS
+Reset Secure Boot Keys option availability (firmware),SBO006.001,PASS,PASS
+Check Password Setup option availability and default state,PSW001.001,PASS,PASS
+Password setting mechanism correctness checking,PSW002.001,PASS,PASS
+Attempt to log in with a correct password,PSW003.001,PASS,PASS
+Attempt to log in with an incorrect password,PSW004.001,PASS,PASS
+Attempt to log in with an incorrect password 3 times,PSW005.001,PASS,PASS
+Attempt to turn off setup password functionality,PSW006.001,PASS,PASS
+Attempt to set non-compilant password,PSW007.001,PASS,PASS
+Attempt to set old password,PSW008.001,PASS,PASS
+Check if ME is statically disabled,MES001.001,PASS,PASS
+Enable USB stack (firmware),USS001.001,FAIL,FAIL
+Disable USB stack (firmware),USS002.001,FAIL,FAIL
+Enable USB Mass Storage (firmware),USS003.001,FAIL,FAIL
+Disable USB Mass Storage (firmware),USS004.001,FAIL,PASS
+SMM BIOS write protection enabling (Ubuntu),SMM001.001,FAIL,PASS
+SMM BIOS write protection disabling (Ubuntu),SMM002.001,FAIL,PASS
+Custom boot menu key,CBK001.001,PASS,PASS
+Custom BIOS Menu Key,CBK002.001,PASS,PASS
+USB devices detected in FW,USB001.001,PASS,PASS
+USB devices detected by OS (Ubuntu),USB001.002,PASS,PASS
+USB devices detected by OS (Windows),USB001.003,PASS,PASS
+USB keyboard detected in FW,USB002.001,PASS,PASS
+USB keyboard in OS (Ubuntu),USB002.002,PASS,PASS
+USB keyboard in OS (Windows),USB002.003,PASS,PASS
+FreeBSD installation and boot,BSD001.001,SKIP,PASS
+Only one iPXE in boot menu,CNB001.002,PASS,PASS
+Boot into UEFI OS (Ubuntu),EFI001.001,PASS,PASS
+Boot into UEFI OS (Windows),EFI001.002,PASS,PASS
+UEFI Shell,USH001.001,PASS,PASS
+NVMe support in firmware,NVM001.001,PASS,PASS
+NVMe support in OS (Ubuntu),NVM001.002 ,PASS,PASS
+NVMe support in OS (Windows),NVM001.003,PASS,PASS
+Wireless card detection (Ubuntu),WLE001.001,PASS,PASS
+Wireless card detection (Windows),WLE001.002,PASS,PASS
+Wi-Fi scanning (Ubuntu),WLE002.001,PASS,PASS
+Wi-Fi scanning (Windows),WLE002.002,PASS,PASS
+Bluetooth scanning (Ubuntu),WLE003.001,PASS,PASS
+Bluetooth scanning (Windows),WLE003.002,PASS,PASS
+External HDMI display in OS (Ubuntu),DSP002.001,PASS,PASS
+External HDMI display in OS (Windows),DSP002.002,PASS,PASS
+External HDMI display in Firmware,DSP002.003,PASS,PASS
+External DP display in OS (Ubuntu),DSP003.001,FAIL,FAIL
+External DP display in OS (Windows),DSP003.002,FAIL,FAIL
```

Comment:
OS doesn't work but firmware works? I would expect the opposite... How is that possible?

Reply:
There may be some weird results here, because I have joined all the tests performed on this platform. external DP worked in RC1 in fw and in OS and then in the future it was only tested in the OS where it failed.

Reply:
How can we keep approaching the validation so lightly? How can we say if something works or not, if we are not sure?

Reply:
It should be re-checked with various monitors. I recall the issue was about the resolution of 1920x1200 vs 1920x1080 (the latter should work).